src/Data/Swagger/Lens.hs

```diff
@@ -62,17 +62,17 @@ makePrisms ''Referenced
 
 _SwaggerItemsArray :: Review (SwaggerItems 'SwaggerKindSchema) [Referenced Schema]
 _SwaggerItemsArray
-  = prism (\x -> SwaggerItemsArray x) $ \x -> case x of
+  = unto (\x -> SwaggerItemsArray x) {- $ \x -> case x of
```

Comment:
Well, the type is `Review` and not `Prism'`, so I fixed the implementation (term) not the spec (type) :)